MessagesView/MessageCollectionViewCell.swift

```diff
@@ -92,6 +102,17 @@ class MessageCollectionViewCell: UICollectionViewCell {
         }
     }
     
+    func addMessageMargin(side: Side, margin: CGFloat) {
```

Comment:
This method is called from outside.
Margin value is configurable.
Treaded similarly as drawTail: